src/test/java/com/uid2/admin/vertx/SiteServiceTest.java

```diff
@@ -694,5 +704,159 @@ void addSiteWithDuplicateDomainNames(Vertx vertx, VertxTestContext testContext)
         });
     }
 
+    @Test
+    void appNameNoSiteId(Vertx vertx, VertxTestContext testContext) {
+        fakeAuth(Role.MAINTAINER);
+        setSites();
+        post(vertx, testContext, "api/site/app_names", "", response -> {
+            assertEquals(400, response.statusCode());
+            assertEquals("must specify site id", response.bodyAsJsonObject().getString("message"));
```

Comment:
I don't think its necessary to test for all error messages that are returned by `getSiteFromParam()` since you are just testing that `handleSiteAppNames()` is called correctly. Still, no harm in adding these tests but not providing a lot of value IMO

Reply:
I just feel it is easy to write and no harm to have it LOL